app/src/main/java/org/oppia/app/help/HelpActivity.kt

```diff
@@ -17,14 +17,26 @@ class HelpActivity : InjectableAppCompatActivity(), RouteToFAQListListener {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    helpActivityPresenter.handleOnCreate()
+    val isFromExploration = intent.getBooleanExtra(
+      BOOL_IS_FROM_EXPLORATION,
+      /* defaultValue= */ false
+    )
+    helpActivityPresenter.handleOnCreate(isFromExploration)
     title = getString(R.string.menu_help)
   }
 
   companion object {
-    fun createHelpActivityIntent(context: Context, profileId: Int?): Intent {
+
+    internal const val BOOL_IS_FROM_EXPLORATION = "BOOL_IS_FROM_EXPLORATION"
```

Comment:
Rename to `BOOL_IS_FROM_EXPLORATION_EXTRA_KEY`

Reply:
Done